dev-requirements.txt

```diff
@@ -16,3 +16,4 @@ Flask>=1.1
 ruff===0.2.2
 # needed for dapr-ext-workflow
 durabletask>=0.1.1a1
+werkzeug>=3.0.3 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
Werkzeug comes in via flask. We just need to make sure we pin a higher Flask version that doesn't include the vulnerable Werkzeug.

Could you do that instead please? The minimum flask version which avoids installing this werkzeug version. Going from 2.2.3 to 3.0.3 is a big jump. A 2.X version would be preferred.